### ISSUE TITLE
gjs: Failed to set text 'Syntax: <Shift>, <Ctrl>, <Alt>, <Super>' from markup due to error parsing markup

### ISSUE THREAD

**Issue by tekstryder** [home-sweet-gnome/dash-to-panel#2012]
- Arch Linux | Kernel 6.6.10
- Gnome-shell | Mutter 46.alpha
- Wayland (`gnome-shell --no-x11`)
- Dash to Panel 59

Forgot to file this a while back. Also occurs on current stable `gnome-shell 45.3`.

```
Jan 03 15:02:36 gjs[263070]: Failed to set text 'Syntax: <Shift>, <Ctrl>, <Alt>, <Super>' from markup due to error parsing markup: Error on line 1 char 56: Element “markup” was closed, but the currently open element is "Super"  
Jan 03 15:02:36 gjs[263070]: Failed to set text 'Syntax: <Shift>, <Ctrl>, <Alt>, <Super>' from markup due to error parsing markup: Error on line 1 char 56: Element “markup” was closed, but the currently open element is “Super”
```
These two errors are emitted when launching Dash to Panel settings UI.

I verified the XML from `BoxIntellihideOptions.ui` and `BoxOverlayShortcut.ui` and can't seem to find an issue.

```
<property name="subtitle" translatable="yes">Syntax: &lt;Shift&gt;, &lt;Ctrl&gt;, &lt;Alt&gt;, &lt;Super&gt;</property>
```

I believe this to be a bug in `gjs` parsing, which I'll file shortly and link here.

**Comment by tekstryder** [comment]
The [Adw.ActionRow.subtitle](https://gjs-docs.gnome.org/adw1~1/adw.actionrow#property-subtitle) property is interpreted as Pango markup. So if you have Pango markup escapes in the XML file, you also need XML escapes.

Attached patch fixes these errors when launching the extension's prefs.

[dash-to-panel-gnome-46alpha-pango-markup.patch.txt](https://github.com/home-sweet-gnome/dash-to-panel/files/13929421/dash-to-panel-gnome-46alpha-pango-markup.patch.txt)

**Comment by tekstryder** [comment]
Referencing https://github.com/home-sweet-gnome/dash-to-panel/pull/2028 as we approach the Gnome 46 release so we don't forget to include this fix. Still required as of shell 46.rc.

**Comment by philippun1** [comment]
@tekstryder I added your patch to #2028

**Comment by tekstryder** [comment]
Verified fixed in v61 tag on Gnome 46.0.

Closing.
